doc/whats_new.rst

```diff
@@ -313,6 +313,11 @@ API changes summary
       :func:`sklearn.model_selection.cross_val_predict`.
       :issue:`2879` by :user:`Stephen Hoover <stephen-hoover>`.
 
+    - Change the default value for ``learning_rate`` in the
+      `sklearn.manifold.TSNE` constructor from 1000 to 200 (with a warning
+      message if not set explicitely). This fixes :issue:`6204` by
+      :user:`John Joseph Valletta <jjvalletta>`.
```

Comment:
We put the issue number of the PR (in this case #8752) and the authors of the PR (in this case probably both @iosonofabio and yourself).